torch.Size([8, 4, 1, 3])
torch.Size([8, 4, 1, 3])
torch.Size([8, 4, 1, 3])
torch.Size([8, 4, 1, 3])
torch.Size([8, 4, 3])
torch.Size([8, 4, 3])
d:\work\work\lib\site-packages\torch\nn\modules\loss.py:536: UserWarning: Using a target size (torch.Size([8, 4, 3])) that is different to the input size (torch.Size([8, 3])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
torch.Size([8, 4, 3])
torch.Size([8, 4, 3])
[34m[1mwandb[39m[22m: [33mWARNING[39m Calling wandb.login() after wandb.init() has no effect.